solr/test-framework/build.gradle

```diff
@@ -43,6 +43,17 @@ dependencies {
   var zkExcludes = {
     exclude group: "org.apache.yetus", module: "audience-annotations"
   }
+  api('org.apache.curator:curator-client', {
+    exclude group: 'org.apache.zookeeper', module: 'zookeeper'
+  })
+  api('org.apache.curator:curator-framework', {
+    exclude group: 'org.apache.zookeeper', module: 'zookeeper'
+  })
+  api('org.apache.curator:curator-test') {
```

Comment:
Thus all solr-test-framework users (plugin users outside of the project) will need curator-test.  Ah; I see this is because ChaosMonkey is here and uses the KillSession thing.  Okay; a little comment might be nice.